tests/system/large/functions/test_remote_function.py

```diff
@@ -1359,6 +1360,51 @@ def square_num(x):
         )
 
 
+@pytest.mark.flaky(retries=2, delay=120)
+def test_remote_function_warns_default_cloud_function_service_account(scalars_dfs):
+    project = "bigframes-dev-perf"
+
+    gcf_service_account = "default"
+
+    rf_session = bigframes.Session(context=bigframes.BigQueryOptions(project=project))
+
+    try:
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
Let's use https://docs.pytest.org/en/stable/reference/reference.html#pytest.warns to make sure a warning _is_ raised.

```suggestion
        with pytest.warns(FutureWarning, match="You have not explicitly set a"):
```